frontend/packages/shared/src/components/Table/Table.tsx

```diff
@@ -56,7 +74,7 @@ export function PaginatedTable<T extends WithId>({
       setCount(data.length);
       setPage(0);
     }
-  }, [data, count]);
+  }, [data, count, setPage]);
```

Comment:
Myös tämän komponentin osalta: pitäisikö vertailla myös taulukon sisältöä eikä pelkästään pituutta?

Reply:
Ei tarvetta enää kun lähtökohtaisesti resetoidaan sivutus aina filtteröinnin muuttuessa

Reply:
Jos näin, niin sitten vissiin tuo `useEffect` on turha?

Reply:
Pidetään toi vanha vielä noin. Teen OTR:n ja VKT:n  osalta oman tiketin ja käyn siinä ne läpi. Siellä on joitain filttereitä, joiden avulla tuo sivutus toimii tällä hetkellä (vaikkakin tuon edge casen osalta bugisesti). Jos tuon vain poistaa mutta ei korjaa sitä tilanhallintaa niin mahdollisesti hajoaa kun jossain näkymässä vaihdetaan sivua mutta sitä ei tabien osalta resetoida -> index out of bounds jos uutta dataa on vähemmän.

Reply:
Mä ajattelisin tän niin että tämän voisi hoitaa paketin versioinnilla => jos ei korjata käyttävää päätä asap niin sitten voivat jäädä käyttämään vanhaa versiota paketista. Toki tää nyt ei ihan semver-konventioita taida noudattaa, mutta se on ehkä pienin murheista.

Reply:
Aa tosi, sharedin puoleltahan sen voi tosiaan poistaa jos ei heti upgradee kaikkialle. Pistetään poistoon ->